app/src/src/screens/createItem/CreateCommunityScreen.jsx

```diff
@@ -199,7 +174,7 @@ class CreateCommunityScreen extends React.Component {
           {/* Render list of poll options and new poll option button if the poll button is selected */}
           {this.state.selectedPostType === 'Poll' && (
             <>
-              {this.state.pollOptions.map((pollText, index) => {
+              {this.state.pollOptions.map((_pollText, index) => {
```

Comment:
Why is there an underscore for the parameter name?

Reply:
it's unused

Reply:
I think you can try turn _pollText into just _ . I've done this in other languages before, but forgot if you can do this in js lol.

Reply:
it worked in js too